lib/Cake/Test/Case/View/Helper/FormHelperTest.php

```diff
@@ -5768,6 +5768,48 @@ public function testSelectMultipleCheckboxes() {
 			'/div'
 		);
 		$this->assertTags($result, $expected);
+
+		$result = $this->Form->select(
+			'Model.multi_field',
+			array('a>b' => 'first', 'a<b' => 'second', 'a"b' => 'third'),
+			array('multiple' => 'checkbox', 'required' => 'required')
+		);
+		$expected = array(
+			'input' => array(
+				'type' => 'hidden', 'name' => 'data[Model][multi_field]', 'value' => '', 'id' => 'ModelMultiField'
+			),
+			array('div' => array('class' => 'checkbox')),
+			array('input' => array(
+				'type' => 'checkbox', 'name' => 'data[Model][multi_field][]',
+				'value' => 'a&gt;b', 'id' => 'ModelMultiFieldAB2',
+				'required' => 'required',
+			)),
```

Comment:
You know what, I see where the confusion came from. I'm using a bootstrap form helper and I'm actually trying to target radios. The bootstrap form helper outputs radios by calling the select method passing multiple=checkbox. I inferred this to mean that cake's formhelper just chose radio as the output type for that situation. I see now that what is actually happening is that it's being generated as checkboxes and then the word checkbox is replaced with radio.

Sorry for confusion. Closing PR